src/index.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+import * as ReactDom from 'react-dom';
+import { ServiceList } from './components/service-list/service.list.component';
+
+ReactDom.render(<ServiceList />, document.getElementById('container'));
```

Comment:
Legg til på slutten for at HMR skal funke riktig

if ((module as any).hot) {
  (module as any).hot.accept();
}